index.js

```diff
@@ -44,3 +44,291 @@ form.addEventListener('submit', (event) => {
+  },
+  {
+    title: 'Multi-Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Modal-pix.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+  {
+    title: 'Facebook 360',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Modal-pix.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+  {
+    title: 'Uber Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'SnapperModals.png',
+    technologies: ['CSS', 'HTML', 'Javascript'],
+    company: 'Canopy',
+    role: 'Back End Dev',
+    year: '2015',
+    liveLink: 'https://coker2019.github.io/Coker_Portfolio/',
+    sourceLink: 'https://github.com/coker2019/Coker_Portfolio',
+  },
+
+];
+
+const projectLists = () => {
+  projectContainer.innerHTML = '';
+  for (let i = 0; i < worksProjects.length; i += 1) {
+    const projectWrapper = document.createElement('div');
+    projectWrapper.setAttribute('class', 'works main');
+    if (i % 2 !== 0) {
+      projectWrapper.setAttribute('class', 'works main main2');
+    }
+
+    const imageWrapper = document.createElement('div');
+    imageWrapper.setAttribute('class', 'cont1');
+    const projectImage = document.createElement('img');
+    projectImage.setAttribute('src', worksProjects[i].imageURL);
+
+    const contBlock = document.createElement('div');
+    contBlock.setAttribute('class', 'cont2');
+
+    const projectName = document.createElement('h2');
+    projectName.textContent = worksProjects[i].name;
+
+    const listItems2 = document.createElement('ul');
+    const list1 = document.createElement('li');
```

Comment:
- [ ] It will be better 😃 if you fix this style here on this card as I see that the icon ```X``` is consistent with the text, so this only happened on this card, so please try to stick to the Figma Design here to make it nice like the Figma Design 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233743239-0a49f120-b868-47cd-95d3-382cba82d3a8.png)  |    ![image](https://user-images.githubusercontent.com/83360636/233743177-8219255e-33ef-453d-b353-56c061131f42.png) |